packages/react-aria-components/src/ListBox.tsx

```diff
@@ -367,8 +367,13 @@ function Option<T>({item}: OptionProps<T>) {
     ref
   );
 
+  let isHoverDisabled = (!states.allowsSelection && !states.hasAction) && 
+                        !(item.props.onHoverStart || item.props.onHoverChange || item.props.onHoverEnd);
```

Comment:
The decision logic is subtle, so if there is a simpler way to write it, please let me know!

Reply:
I was going to say this was fine, but since the second conditional makes it so `isHovered` is updated if the user passes any hover event handlers it now feels like we are straying into the same territory as this past PR: https://github.com/adobe/react-spectrum/pull/5559#issuecomment-1856352425. We'll have to discuss this notion of when the hover state is actually updated and make sure it doesn't cause a breaking change to people's current implementation (aka they might be relying on `isHovered` only appearing on interactive elements). We'll want to make sure this is consistent across other collection component items as well.

Reply:
Thanks for the feedback. I'll take a look at the PR's comments on the link and see if I can figure out how to implement it.

Reply:
thanks for taking a look, let us know if you have any other questions as you dig. I'll see if the team can discuss this soon and come up with a path forward as well

Reply:
talked with the team, we wanna keep the current behavior of "if a option doesn't have an action or allows selection, it is non-interactive" so lets keep the old ` isDisabled: !states.allowsSelection && !states.hasAction` logic

Reply:
Okay, I think that would be better.
I'll try to modify it that way. thanks.

Reply:
@LFDanLu 
I have fixed it!
[fix: modified to original logic](https://github.com/adobe/react-spectrum/pull/5777/commits/d844f2fa95b56e19295dce06a727608d8e631055)